### PR TITLE
3D Heatmap Workshop

### PR DESCRIPTION
**Pull request by shivesh01**
I have completed all the requested changes in the workshop. Please review it. I think it is ready to merge.

### CODE REVIEW

File: workshops/3D Heatmap/README.md [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+fig = plt.figure()
+ax = plt.axes(projection='3d')
+plt.show()
+```
+![3D Plane](https://cloud-a08hob7s0.vercel.app/1workshop2.gif)
+
+# 2. Plotting Graph
+
+Plotting 3-D a Graph
+```python
```
Comment (philip-hub) [inline workshops/3D Heatmap/README.md:86]: why are you importing again this may be kind of confusing

File: workshops/3D Heatmap/README.md [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+
+
+# Keep Hacking!
+Mountain Bruno [Code](https://repl.it/@ShiveshSingh/Mtbrunoplot)
+
+Surface Plot [Code](https://repl.it/@ShiveshSingh/Surface-Plot-3D#main.py)
+
+Heat Map Animation [Code](https://repl.it/@ShiveshSingh/HeatmapAnimation)
+
+
```
Comment (philip-hub) [inline workshops/3D Heatmap/README.md]: My final thoughts

- Fix the second section called plotting a graph by importing numpy and by getting rid of the instructions in the code which causes syntax errors.
- Fix the continue hacking section or explain what I have to do to see it properly.
- Be a little more specific on complex line function especially with a complex library like pandas and the 3D mapping mostly in section 3

File: workshops/3D Heatmap/README.md [hunk 3]
```diff
@@ -0,0 +1,202 @@
+---
```
Comment (philip-hub) [inline workshops/3D Heatmap/README.md:1]: Good job I left some various comments. Its not ready to be a workshop yet, but if change a few things it probably could be a workshop.

File: workshops/3D Heatmap/README.md (hunk 1 above)
Comment (shivesh01) [reply workshops/3D Heatmap/README.md:86]: 👍🏻

File: workshops/3D Heatmap/README.md [hunk 4]
```diff
@@ -0,0 +1,202 @@
...
+- Repl is an online and instant development environment to learn, build, collaborate, and host your project. Show you don’t have to waste time while setting up a development environment. Creating an account on repl is super easy.
+
+[Create a repl](https://repl.it/signup)
+
+
+![repl website](https://cloud-73h0sldam.vercel.app/0screenshot_2020-12-25_at_23.03.53.png)
+
+- create a repl by clicking **+ New repl** and choose programming language **PYTHON** and named it as you want and click **create repl**
+
+- Download the datasets from a simple click and Drag and drop them to the folder in which you are working on repl.
```
[the remark this answers is not in the document]
Comment (shivesh01) [reply workshops/3D Heatmap/README.md]: Done👍🏻

File: workshops/3D Heatmap/README.md [hunk 5]
```diff
@@ -0,0 +1,202 @@
...
+
+```python
+from mpl_toolkits.mplot3d import Axes3D
+import matplotlib.pyplot as plt
+import pandas as pd
+import seaborn as sns
+```
+
+Get the data (CSV file) from the file
+volcano [dataset](https://cloud-8ycpvzexa.vercel.app/0volcano.csv)
```
[the remark this answers is not in the document]
Comment (shivesh01) [reply workshops/3D Heatmap/README.md:136]: Done

File: workshops/3D Heatmap/README.md [hunk 6]
```diff
@@ -0,0 +1,202 @@
...
+### Marvelous! You have created your first 3D Heat Map plot ###
+
+
+# voilà!
+
+
+**Congratulations!✌️ you have completed the workshop. Share it with your friends and family.
+Share with everyone🤗✌️🥳👏🏅🌇🎊**
+
+![congratulations g.i.f](https://cloud-1th3ydnib.vercel.app/2workshop_happy.gif)
```
[the remark this answers is not in the document]
Comment (shivesh01) [reply workshops/3D Heatmap/README.md:192]: Thanks :)

File: workshops/3D Heatmap/README.md (hunk 2 above)
Comment (shivesh01) [reply workshops/3D Heatmap/README.md]: Done

File: workshops/3D Heatmap/README.md [hunk 7]
```diff
@@ -0,0 +1,202 @@
...
+
+
+
+
+# Keep Hacking!
+Mountain Bruno [Code](https://repl.it/@ShiveshSingh/Mtbrunoplot)
```
[the remark this answers is not in the document]
Comment (shivesh01) [reply workshops/3D Heatmap/README.md:193]: Done

File: workshops/3D Heatmap/README.md [hunk 8]
```diff
@@ -0,0 +1,202 @@
...
+ax = plt.axes(projection='3d')
+plt.show()
+```
+![3D Plane](https://cloud-a08hob7s0.vercel.app/1workshop2.gif)
+
+# 2. Plotting Graph
+
+Plotting 3-D a Graph
+```python
+from mpl_toolkits.mplot3d import Axes3D
```
[the remark this answers is not in the document]
Comment (shivesh01) [reply workshops/3D Heatmap/README.md:86]: Done

File: workshops/3D Heatmap/README.md (hunk 3 above)
Comment (shivesh01) [reply workshops/3D Heatmap/README.md:1]: Thanks, for helping me out with all changes done.

File: workshops/3D Heatmap/README.md [hunk 9]
```diff
@@ -0,0 +1,202 @@
...
+import pandas as pd
+import seaborn as sns
+```
+The most basic three-dimensional plot is a line or collection of scatter plot created from sets of (x, y, z) triples.
+
+```
+ax = plt.axes(projection='3d')
+z = np.linspace(0, 30, 100)
+
+Initialise x is array of np.sin(z), and similarly y is a array of np.cos(z)
```
[the remark this answers is not in the document]
Comment (shivesh01) [reply workshops/3D Heatmap/README.md]: it's an array

File: workshops/3D Heatmap/README.md [hunk 10]
```diff
@@ -0,0 +1,202 @@
...
+```
+Add a color bar that maps values to colors. The point height on the mountain increase, the color value will shift from blue to red.
+
+```python
+surf = ax.plot_trisurf(df['Y'], df['X'], df['Z'], cmap=plt.cm.jet, linewidth=0.2)
+fig.colorbar(surf, shrink=0.5, aspect=5)
+```
+We will use this function to display the plot on our computer screen.
+```
+plt.show()
```
[the remark this answers is not in the document]
Comment (shivesh01) [reply workshops/3D Heatmap/README.md:223]: Thanks man

File: workshops/3D Heatmap/README.md [hunk 11]
```diff
@@ -0,0 +1,273 @@
...
+
+# Getting started
+
+Today, we'll be using repl.it to write out doe in. It is an online, instant development environment to learn, build, collaborate, and host your project. Show you don’t have to waste time while setting up a development environment. 
+
+[Sign up and then create a repl here.](https://repl.it/signup)
+
+![repl website](https://cloud-73h0sldam.vercel.app/0screenshot_2020-12-25_at_23.03.53.png)
+
+- You can fork our repl which contains a volcano.csv file\
```
Comment (sampoder) [inline workshops/3D Heatmap/README.md:26]:
```suggestion
To help get you started, you can fork our repl which contains a volcano.csv file. Click [here](https://repl.it/@ShiveshSingh/3DHeatmapWorkshop) to load up that repl for forking.
```

File: workshops/3D Heatmap/README.md [hunk 12]
```diff
@@ -0,0 +1,273 @@
...
+Fantastic! You have created your first 3D Heatmap plot.
+
+# Voilà!
+
+Congratulations!✌️ you have completed the workshop. Share it with your friends and family.
+Share with everyone🤗✌️🥳👏🏅🌇🎊
+
+![congratulations g.i.f](https://cloud-1th3ydnib.vercel.app/2workshop_happy.gif)
+
+# Keep Hacking!
```
Comment (sampoder) [inline workshops/3D Heatmap/README.md:266]: Let's make these more human, give each one a paragraph. View Tunes for an example.

**Review by sampoder — COMMENTED** [review]
View review comments and also please add code formatting everywhere.

Thank you so much,

Sam 🚀 😄

File: workshops/3D Heatmap/README.md (hunk 11 above)
Comment (shivesh01) [reply workshops/3D Heatmap/README.md:26]: Done

**Pull request closed without merging**